docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -67,6 +67,19 @@ You can upload a custom checkout to your store's server using WebDAV. The instru
 
 ## Installing custom checkouts  
 
+<div class="HubBlock--callout">
```

Comment:
We no longer need the HTML wrapper with Stoplight Platform, though it doesn't hurt anything.